spec/acceptance/keystore_failed_password_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper_acceptance'
+
+describe 'managing java keystores without a correct password', :unless => UNSUPPORTED_PLATFORMS.include?(fact('operatingsystem')) do
+  case fact('osfamily')
+  when "Solaris"
+    keytool_path = '/usr/java/bin/'
+    resource_path = "['/usr/java/bin/','/opt/puppet/bin/']"
+  when "AIX"
+    keytool_path = '/usr/java6/bin/'
+    resource_path = "['/usr/java6/bin/','/usr/bin/']"
+  else
+    resource_path = "undef"
+  end
+  it 'creates a keystore' do
+    pp = <<-EOS
+      java_ks { 'puppetca:keystore':
+        ensure       => latest,
+        certificate  => "/tmp/ca.pem",
+        target       => '/etc/keystore_failed_password.ts',
+        password     => 'coraline',
+        trustcacerts => true,
+        path         => #{resource_path},
+    }
+    EOS
+    apply_manifest(pp, :catch_failures => true)
+  end
+
+  it 'verifies the keystore' do
+    shell("#{keytool_path}keytool -list -v -keystore /etc/keystore.ts -storepass coraline") do |r|
```

Comment:
Should this (and the other 3 occurances) be `-keystore /etc/keystore_failed_password.ts` to fix the failing acceptance tests?
```
keytool error: java.lang.Exception: Keystore file does not exist: /etc/truststore.ts
java.lang.Exception: Keystore file does not exist: /etc/truststore.ts
```